src/main.py

```diff
@@ -0,0 +1,8 @@
+# Main entry point to run the recommendation pipeline
```

Comment:
nit: this comment is nonsense, we should probably just remove it.

Reply:
Removed it